dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -91,5 +95,100 @@ interface RequestApi {
     fun getAggregatedDataRequests(
         @RequestParam identifierValue: String? = null,
         @RequestParam dataTypes: Set<DataTypeEnum>? = null,
-    ): List<AggregatedDataRequest>
+        @RequestParam reportingPeriod: String? = null,
+    ): ResponseEntity<List<AggregatedDataRequest>>
+
+    /**
+     * A method to post a single request to Dataland.
+     * @param singleDataRequest includes necessary info for the single request
+     * @return response after posting a single data request to Dataland
+     *
+     */
+    @Operation(
+        summary = "Send a single request",
+        description = "A single of data requests for specific frameworks and companies is being sent.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully processed a single data request."),
+        ],
+    )
+    @PostMapping(
+        value = ["/single"],
+        consumes = ["application/json"],
+        produces = ["application/json"],
+    )
+    @PreAuthorize("hasRole('ROLE_USER')")
+    fun postSingleDataRequest(
+        @Valid @RequestBody
```

Comment:
Florian raised an interesting question coming from his e-mail story. Should it be possible to request more than one framework / one reporting period in a single request?

Reply:
Yes, but the question is should it be possible or should it rather be that in a single data request only one framework for one reportin period can be requested, as in the designs for the email that is being sent only one reporting period is displayed.

Reply:
Ah now I see what you mean. I also thought about this, but we decided at the end to trust the FIGMA. I'm not sure what to do with this, what do you think @emanueldfine ?

Reply:
Maybe we need feedback here in tomorrows daily

Reply:
ok was clarified in the daily
if @fsoffel, @emanueldfine, @PaulChristiansen, @Worben you are fine with it, i will close it